packages/server/modules/workspaces/services/workspaceProjects.ts

```diff
@@ -0,0 +1,31 @@
+import { StreamRecord } from '@/modules/core/helpers/types'
```

Comment:
Do `workspaceProjects` and `workspaceProjectRoleCreation` really need to be 2 different services? It's kind of subjective, so if you think they do, then we can leave it at that, but I'd like to avoid having dozens of tiny services that only have 1 function in them